plugins/Phone.py

```diff
@@ -287,14 +287,15 @@ class father:
     def test_IT(self):
         p = Phone(None)
         class _config:
-            options = {"country": "IT", "phone_code": "39", "phone_len": [6, 11], "phone_len_short": [2, 3, 4], "phone_international": "00", "phone_format": r"^[+]%s[- ]*[038][0-9]+(?:[- ][0-9]+)?(?:(?:[- ][0-9]+)|$)$"}
+            options = {"country": "IT", "phone_code": "39", "phone_len": [6, 11], "phone_len_short": [2, 3, 4], "phone_international": "00", "phone_format": r"^[+]%s[- ]*[03][0-9]+(?:[- ][0-9]+)?(?:(?:[- ][0-9]+)|$)$"}
         class father:
             config = _config()
         p.father = father()
         p.init(None)
 
         for (bad, good) in (
-            (u"800 123", u"+39 800 123"),
+            # Use phone:IT for toll-free numbers
+            # (u"800 123", u"+39 800 123"),
```

Comment:
toll-free are callable from aboard ? It is not the case in France, so we keep it in local form.